### PR TITLE
Import RTV report after creating 

### PR DESCRIPTION
**Pull request by aaronschachter**
### What's this PR do?

This pull request refactors the import RTV console command (`app/Console/Commands/ImportRockTheVoteReport`) introduced in #127 as a queued job (moving it to (`app/Jobs/ImportRockTheVoteReport`), and dispatches the job after creating a report via web (#128). 

It also adds some extra meta fields to our `RockTheVoteReport` model to store the progress of report generation locally as the import job checks the RTV API for the report status, and modifies the `show` view to display the model instead of making an API request. Refs https://github.com/DoSomething/chompy/pull/128#discussion_r389750611

Screen Shot 2020-03-12 at 7 20 05 AM

### How should this be reviewed?

👀 

### Any background context you want to provide?

This could really use some test coverage! I've been testing locally by creating real reports which is not ideal. I'm going to add coverage in a new PR because it requires adding `browser-kit-testing` (see https://github.com/DoSomething/chompy/pull/130#issuecomment-597918994) and looking to keep this PR easier to review. 

### Relevant tickets

References [Pivotal #171656723](https://www.pivotaltracker.com/story/show/171656723).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: app/Jobs/ImportRockTheVoteReport.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        $this->report->status = $status;
+        $this->report->row_count = $response->record_count;
+        $this->report->current_index = $response->current_index;
+
+        info('Report status is ' . $status);
+
+        if ($status !== 'complete') {
+            $this->report->save();
+
+            return self::dispatch($this->user, $this->report)->delay(now()->addMinutes(2));
```
Comment (katiecrane) [inline app/Jobs/ImportRockTheVoteReport.php]: So if the report is not complete, we check on it every 2 minutes until it is?

Comment (aaronschachter) [reply app/Jobs/ImportRockTheVoteReport.php]: Yep! 🤖

File: app/Jobs/ImportRockTheVoteReport.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        $now = Carbon::now();
+        $path = 'uploads/' . ImportType::$rockTheVote . '-report-' . $reportId . '-' .$now . '.csv';
+
+        Storage::put($path, app(RockTheVote::class)->getReportByUrl($response->download_url));
+
+        info('Downloaded report '.$reportId);
+
+        ImportFileRecords::dispatch($this->user, $path, ImportType::$rockTheVote, ['report_id' => $reportId])->delay(now()->addSeconds(3));
+
+        $this->report->dispatched_at = $now;
```
Comment (katiecrane) [inline app/Jobs/ImportRockTheVoteReport.php]: So `dispatched_at` is when the report was downloaded?

Comment (aaronschachter) [reply app/Jobs/ImportRockTheVoteReport.php]: Yes, and dispatched for import (by calling `ImportFileRecords` with our downloaded file path).

Comment (katiecrane) [reply app/Jobs/ImportRockTheVoteReport.php]: ah got it! was wondering about the naming

**Pull request merged**